tests/run_test.sh

```diff
@@ -246,8 +246,11 @@ fi
 if [[ $skip_check_results = false ]]; then
   check_results
 else
-  echo "Test ${TEST_NR} ${TEST_NAME} RUN_SUCCESS" >  ${REGRESSIONTEST_LOG}
-  echo;echo;echo                                  >> ${REGRESSIONTEST_LOG}
+  echo                                               >> ${REGRESSIONTEST_LOG}
+  grep "The total amount of wall time" ${RUNDIR}/out >> ${REGRESSIONTEST_LOG}
+  echo                                               >> ${REGRESSIONTEST_LOG}
```

Comment:
Can we also grep the memory usage "The maximum resident set size (KB)" from the ${RUNDIR}/out file? I'd suggest to add this memory info to the regular RT log for each test too.

Reply:
Added in https://github.com/ufs-community/ufs-weather-model/pull/967/commits/97257bcc79436a53667336b789fe2956fe960f08